bitnami/metallb/templates/tests/test-connection.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "metallb.fullname" . }}-test-connection"
+  labels: {{- include "metallb.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
```

```suggestion
      image: bitnami/minideb:buster
```

We use minideb for this kind of actions